app/src/userpages/components/ProductsPage/index.jsx

```diff
@@ -70,7 +70,13 @@ const ProductsPage = () => {
     } = useAllDataUnionStats()
 
     useEffect(() => {
-        dispatch(getMyProducts(filter))
+        // Modify filter to include only normal products
+        const finalFilter = {
+            ...filter,
+            // key: 'type',
+            // value: 'normal',
+        }
+        dispatch(getMyProducts(finalFilter))
```

Comment:
It does not look like we're modding the filter here. Is it necessary?

Reply:
Yeah, true. I left it there hanging depending on if we want to filter out data unions from the product tab but maybe we leave it like it was so I'll remove the commented code.

Reply:
Ah, ok. Who is to say if we want that? Also, does it mean both products and data unions are displayed on the product listing page, and on the data union listing page we see only data unions?

Reply:
That's what it would mean yeah. But Juha had a good point at https://github.com/streamr-dev/core-frontend/pull/1218#issuecomment-824752598